src/main/webapp/scripts/session-script.js

```diff
@@ -231,39 +207,65 @@ function buildAttendeeDiv(nameOfAttendee, controller) {
  * toggle, their controller status is revoked and the server is updated
  * with information on the new controller.
  * @param {MouseEvent} event the event that captures what was clicked on
- * @param {string} controller name of the controller of the session
+ * @param {string} currentControllerName 
+ *    name of the controller of the session
  */
-function changeControllerTo(event, controller) {
-  if (urlParameters.get(URL_PARAM_KEY.SCREEN_NAME) === controller) {
-    try {
-      serverClient.changeControllerTo(/**newControllerName=*/
-          event.target.parentElement.querySelector('h3').id);
-    } catch (e) {
-      window.alert('No contact with the server!');
-    }
-  }
+function changeControllerTo(event, currentControllerName) {
+  if (urlParameters.get(sessionScriptConstants.URL_PARAM_KEY.SCREEN_NAME) 
+      === currentControllerName) {
+        try {
+          serverClient.changeControllerTo(/**newControllerName=*/
+              event.target.parentElement.querySelector('h3').id);
+        } catch (e) {
+          window.alert('No contact with the server!');
```

Comment:
Again, do they reload the page? What should a user do if they see this alert?

Reply:
they'd have to wait for updateUI to run again

Reply:
in this case they can try changing again